apis/v1alpha1/httproute_types.go

```diff
@@ -534,27 +537,31 @@ type HTTPRouteForwardTo struct {
 	//
 	Port PortNumber `json:"port"`
 
-	// Weight specifies the proportion of traffic forwarded to the backend
-	// referenced by the ServiceName or BackendRef field. computed as
-	// weight/(sum of all weights in this ForwardTo list). Weight is not a
-	// percentage and the sum of weights does not need to equal 100. If only one
-	// backend is specified, 100% of the traffic is forwarded to that backend.
-	// If weight is set to 0, no traffic should be forwarded for this entry.
-	// If unspecified, weight defaults to 1.
+	// Weight specifies the proportion of HTTP requests forwarded to the backend
+	// referenced by the ServiceName or BackendRef field. This is computed as
+	// weight/(sum of all weights in this ForwardTo list). For non-zero values,
+	// there may be some epsilon from the exact proportion defined here
+	// depending on the precision an implementation supports. Weight is not a
+	// percentage and the sum of weights does not need to equal 100.
+	//
+	// If only one backend is specified and it has a weight greater than 0, 100%
+	// of the traffic is forwarded to that backend. If weight is set to 0, no
+	// traffic should be forwarded for this entry. If unspecified, weight
+	// defaults to 1.
 	//
 	// Support: Core
 	//
 	// +kubebuilder:default=1
 	// +kubebuilder:validation:Minimum=0
-	// +kubebuilder:validation:Maximum=10000
+	// +kubebuilder:validation:Maximum=1000000
 	Weight int32 `json:"weight,omitempty"`
 
 	// Filters defined at this-level should be executed if and only if the
 	// request is being forwarded to the backend defined here.
 	//
-	// Conformance: Filtering support, including core filters, is NOT guaranteed
-	// at this-level. Use Filters in HTTPRouteRule for portable filters across
-	// implementations.
+	// Conformance: Support for filters defined at this level is NOT guaranteed.
```

Comment:
I find it confusing that "Conformance" is defined for `Filters` but nowhere else in the APIs. How should the combination of "Conformance" and "Support" be determined. Should "Conformance" be removed and update "Support: Custom" as follows:
```
	// Support: Custom (For broader support of filters, use the Filters field in HTTPRouteRule)
```

Reply:
Updated, thanks!